README.md

```diff
@@ -127,7 +127,21 @@ public class MainActivity extends ReactActivity {
 
 Add `Indy.framework` as dependency into your project. Beware that the library needs to be compiled with the same version of Swift as your project.
 
-React Native wrapper for iOS is written in Swift so you need to Create briding header. In Xcode app menu go to File -> New -> File... Select Swift File from dialog window and click Next and Finish. Xcode shoud ask you "Would you like to configure an Objective-C briding header?" then select Create Bridinging Header.
+Open `.xcworkspace` file in Xcode and in your project settings, tab General, section Frameworks, Libraries, and Embedded Content, click on plus. Then select Add Other -> Add files... and navigate to `Indy.framework` file on your disk.
+
+Create `Frameworks` folder in your project's `ios/Pods` directory and copy `Indy.frameworks` also there. (I think there should be a way how to avoid keeping `Indy.framework` in two places, but this way at least worked for me).
```

Comment:
I copied `Indy.framework` to the `ios/Pods/Frameworks` directory, added that to `.gitignore.` and added that one in Xcode. This way you don't need the `Indy.framework` two times, but only one

Reply:
Good tip. I haven't tried that, but I trust you 😄

Reply:
Trust but verify 😉

https://github.com/animo/aries-mobile-agent-react-native/tree/develop/ios/Pods/Frameworks